test/src/discover_tasks_test.dart

```diff
@@ -147,6 +148,40 @@ main() {
     });
   });
 
+  group('execute returns value', () {
+    grinderTest('task from method', () {}, (MockGrinderContext ctx) {
+      var annotated =
+          discoveryGood.discoverDeclaration(goodLib.declarations[#method], {});
+      var task = annotated.task;
+      expect(task.name, 'method');
```

Comment:
There are already tests for the names of discovered tasks, so can we remove those here?